aiida_orca/parsers/__init__.py

```diff
@@ -70,24 +74,18 @@ def _remove_nan(parsed_dictionary: dict) -> dict:
 
         output_dict = _remove_nan(parsed_dict)
 
-        # keywords = output_dict['metadata']['keywords']
-
         if parsed_dict.get('optdone', False):
-            with out_folder.open(fname_relaxed) as handler:
-                relaxed_structure = StructureData(pymatgen_molecule=Molecule.from_file(handler.name))
+            with out_folder.open(fname_relaxed, 'rb') as handle:
+                with tempfile.NamedTemporaryFile('w+b', suffix=pathlib.Path(fname_relaxed).suffix) as tmpfile:
+                    shutil.copyfileobj(handle, tmpfile)
+                    tmpfile.flush()
+                    relaxed_structure = StructureData(pymatgen_molecule=Molecule.from_file(tmpfile.name))
             self.out('relaxed_structure', relaxed_structure)
-            # relaxation_trajectory = SinglefileData(
-            #     file=os.path.join(out_folder._repository._get_base_folder().abspath, fname_traj)  #pylint: disable=protected-access
-            # )
-            # self.out('relaxation_trajectory', relaxation_trajectory)
 
-        pt = PeriodicTable()  #pylint: disable=invalid-name
+        pt = PeriodicTable()  # pylint: disable=invalid-name
 
-        output_dict['elements'] = [pt.element[Z] for Z in output_dict['atomnos'].tolist()]
+        output_dict['elements'] = [pt.element[Z] for Z in output_dict['atomnos']]
```

Comment:
Why was this changed? Would this change work even if we ran a calculation on a single atom?

Reply:
Think it shouldn't matter because it is a `numpy.ndarray` which is iterable so I don't think conversion to a list is necessary. But I can keep it, it is not a critical change.